hokusai/commands/retag.py

```diff
@@ -0,0 +1,16 @@
+from hokusai.lib.command import command
+from hokusai.lib.common import print_green, print_yellow
+from hokusai.services.ecr import ECR, ClientError
+
+@command()
+def retag(tag_to_change, tag_to_match):
+  ecr = ECR()
+
+  if not ecr.project_repo_exists():
+    raise HokusaiError("Project repo does not exist.  Aborting.")
+
+  try:
+    ecr.retag(tag_to_match, tag_to_change)
+    print_green("Updated ECR '%s' tag to point to the image that '%s' tag points to." % (tag_to_change, tag_to_match), newline_after=True)
+  except (ValueError, ClientError) as e:
+    print_yellow("WARNING: Updating ECR tag failed due to the error: '%s'" % str(e), newline_before=True, newline_after=True)
```

Comment:
Curious why you chose to flag this as a "Warning" and not `print_red("Error: ...)` here?

Reply:
That's b/c I copied it from [service/deployment.py](https://github.com/artsy/hokusai/blob/ecf03293b62eafcf535e860c0474bab8dc141097/hokusai/services/deployment.py#L162). Over here, since ECR tagging is the sole function, I agree it makes more sense to print read. Updated. Thanks.